test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGetPackageManagerTests.cs

```diff
@@ -3494,6 +3494,7 @@ public async Task TestPacManOpenReadmeFile()
                 Assert.Equal(packageIdentity, packagesInPackagesConfig[0].PackageIdentity);
                 Assert.Equal(msBuildNuGetProject.ProjectSystem.TargetFramework, packagesInPackagesConfig[0].TargetFramework);
                 Assert.Equal(1, testNuGetProjectContext.TestExecutionContext.FilesOpened.Count);
+                Assert.Equal(0, testNuGetProjectContext.TestExecutionContext.MarkdownFilesOpened.Count);
```

Comment:
It's good to have the existing tests validate the expected behavior when it comes to the markdown files. In this case, none should be opened which this is validating.

There should be a new test that validates that the readme.md file will be opened if no readme.txt is present.